plugins/org.jkiss.dbeaver.ext.cubrid.ui/src/org/jkiss/dbeaver/ext/cubrid/ui/views/PrefPageCubrid.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.ui.views;
+
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Group;
+import org.jkiss.dbeaver.ext.cubrid.CubridConstants;
+import org.jkiss.dbeaver.ext.cubrid.ui.internal.CubridMessages;
+import org.jkiss.dbeaver.model.DBPDataSourceContainer;
+import org.jkiss.dbeaver.model.preferences.DBPPreferenceStore;
+import org.jkiss.dbeaver.runtime.DBWorkbench;
+import org.jkiss.dbeaver.ui.UIUtils;
+import org.jkiss.dbeaver.ui.preferences.PreferenceStoreDelegate;
+import org.jkiss.dbeaver.ui.preferences.TargetPrefPage;
+import org.jkiss.dbeaver.utils.PrefUtils;
+
+public class PrefPageCubrid extends TargetPrefPage
+{
+	public static final String PAGE_ID = "org.jkiss.dbeaver.preferences.cubrid.general"; //$NON-NLS-1$
+	private Button trace;
+	private Button info;
+	private Button allInfo;
+	 public PrefPageCubrid()
+	    {
+	        super();
+	        setPreferenceStore(new PreferenceStoreDelegate(DBWorkbench.getPlatform().getPreferenceStore()));
+	    }
+
+	@Override
+	protected boolean hasDataSourceSpecificOptions(DBPDataSourceContainer dsContainer) {
+		// TODO Auto-generated method stub
+		return false;
+	}
+
+	@Override
+	protected boolean supportsDataSourceSpecificOptions() {
+		return true;
+	}
+
+	@Override
+	protected void loadPreferences(DBPPreferenceStore store) {
+		trace.setSelection(store.getBoolean(CubridConstants.STATISTIC_TRACE));
+		if(store.getString(CubridConstants.STATISTIC).equals(CubridConstants.STATISTIC_INFO)) {
+			info.setSelection(true);
+		}else if(store.getString(CubridConstants.STATISTIC).equals(CubridConstants.STATISTIC_ALL_INFO)) {
+			allInfo.setSelection(true);
+		}
```

Comment:
Could be a direct assignment.

```suggestion
		info.setSelection(store.getString(CubridConstants.STATISTIC).equals(CubridConstants.STATISTIC_INFO));
		allInfo.setSelection(store.getString(CubridConstants.STATISTIC).equals(CubridConstants.STATISTIC_ALL_INFO));
```